### ISSUE TITLE
Bot doens't react to waifu above the min value *buttons are enabled so invalid set up*

### ISSUE THREAD

**Issue by enzoIto** [vivinano/MudaeAutoBot#188]
Did a few tryouts and bot rolled a megumin (1000+ kakera) and not reacted at all (the min value was at 400 kakeras).

**Comment by vivinano** [comment]
Did the bot have a claim available when the character popped up ?

**Comment by enzoIto** [comment]
Yup. Also, all last claim attempts failed because the bot never reacted. Tried switching to custom emoji to see if I could get it to work, but the error persists

**Comment by vivinano** [comment]
Screen shots with the logs enabled?

**Comment by enzoIto** [comment]
![2022-07-29_11-13](https://user-images.githubusercontent.com/106115847/181779467-cccdd637-6389-428c-9472-70dbc35cd5fc.png)
![2022-07-29_11-09](https://user-images.githubusercontent.com/106115847/181779552-2535ff93-a7f0-412c-a430-22cbe85cf855.png)
![2022-07-29_11-14](https://user-images.githubusercontent.com/106115847/181779562-394e937c-78b8-46a3-8225-eaaf517dd7c1.png)

Reseted the custom emojis to default and did a test with min value at 10. Also, I noticed that the bot didn't react to kakeras.

**Comment by enzoIto** [comment]
![image](https://user-images.githubusercontent.com/106115847/181780894-7447a752-fc50-406d-8284-68595baaedac.png)
![2022-07-29_11-09](https://user-images.git
![2022-07-29_11-14](https://user-images.githubusercontent.com/106115847/181780937-77185fa4-b05e-4be1-9cc0-dec2c335ce50.png)
hubusercontent.com/106115847/181780915-0cc12096-464e-4bff-85fa-ee2c3f2b668e.png)
![image](https://user-images.githubusercontent.com/106115847/181781100-c372d416-4b17-4bfd-89a2-2027e046c3c1.png)


Reseted the custom emojis and did a test. Also, noticed that the bot didn't react to kakeras.

**Comment by enzoIto** [comment]
![2022-07-29_11-09](https://user-images.githubusercontent.com/106115847/181781354-e1606217-5b96-4a54-a12d-36cdbc76b8ea.png)

bot seetings. min kak was at 10.

**Comment by vivinano** [comment]
I see in the picture that those are not reactions but buttons which require you to fill out extra settings for it to work

**Comment by enzoIto** [comment]
Ohh now I see. So to the bot works it needs to disable those buttons? Like $togglereact 1

**Comment by vivinano** [comment]
I mean it work for either but for buttons you need to have specific settings

**Comment by vivinano** [comment]
Gonna close the issue with invalid settings label as this is an issue with assuming reactions and buttons are the same but they are not and require different set up
